routes/bears.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+const Bear = require('../models/bear');
+const mongoose = require('mongoose');
+
+//mongoose.connect('mongodb://localhost/bear', { useNewUrlParser: true });
+mongoose
+	.connect('mongodb+srv://greenlad:8564@greenlad@cluster0-jonki.mongodb.net/test?retryWrites=true', { useNewUrlParser: true })
+	.then(() => console.log('Conneting to mongoDB'))
+	.catch((err) => console.log(err.message));
+
+//get verb for Bear
+router.get('/api/bears', async (req, res) => {
+	const bears = await Bear.find();
+
+	res.json(bears);
+});
+
+//GetByID verb for bears
+router.get('/api/bear/:id', async (req, res) => {
+	const bear = await Bear.find({ _id: req.params.id });
+	if (!bear.length) return res.status(404).send(`There is no Bear with the ID: ${req.params.id}`);
+	res.json(bear);
+});
+
+//Getting bears by Bearname
+router.get('/api/bears/name/:bear_name', async (req, res) => {
```

Comment:
Are you sure the bear name is unique?

Reply:
the bear names wouldn't have to be unique. the aim is to get bears with the same name